src/data/pokeball.ts

```diff
@@ -117,5 +145,40 @@ export function doPokeballBounceAnim(scene: BattleScene, pokeball: Phaser.GameOb
     });
   };
 
-  doBounce();
+  const doCritShake = () => {
+    scene.tweens.add({
+      targets: pokeball,
+      x: x2,
+      duration: 125,
+      ease: "Linear",
+      onComplete: () => {
+        scene.tweens.add({
+          targets: pokeball,
+          x: x1,
+          duration: 125,
+          ease: "Linear",
+          onComplete: () => {
+            critShakes--;
+            if (critShakes > 0) {
+              doCritShake();
+            } else {
+              scene.tweens.add({
+                targets: pokeball,
+                x: x0,
+                duration: 60,
+                ease: "Linear",
+                onComplete: () => doBounce()
```

Comment:
```suggestion
                onComplete: () => scene.time.delayedCall(500, doBounce)
```
A slight delay to make the critical capture animation a bit more pronounced